guides/source/active_record_validations.md

```diff
@@ -777,7 +777,21 @@ Topic.create(title: nil).valid? # => true
 As you've already seen, the `:message` option lets you specify the message that
 will be added to the `errors` collection when validation fails. When this
 option is not used, Active Record will use the respective default error message
-for each validation helper.
+for each validation helper. The `:message` option accepts a `String` or `Proc`.
+
+```ruby
+class Person < ActiveRecord::Base
+  # Hard-coded message
+  validates :name, presence: { message: "must be given please" }
+
+  # Message with dynamic attribute value
+  validates :age, numericality: { message: "%{value} seems wrong" }
+
+  # Proc result as a message
+  validates :username,
+    uniqueness: { message: -> "taken! Try again #{Time.zone.tomorrow}" }
```

Comment:
Don't we need brackets for the lambda? Also does the lambda get passed any values like the object being validated against? If so can you add another example to show that?